test/hotspot/jtreg/vmTestbase/nsk/stress/jni/libjnistress003.cpp

```diff
@@ -40,225 +40,230 @@ extern "C" {
-    jfloat *floatBuf;
-    jdouble *doubleBuf;
-    jobjectArray objectsArray;
+  jarray *arrayArray;
+  jboolean *boolBuf;
+  jbyte *byteBuf;
+  jchar *charBuf;
+  jshort *shortBuf;
+  jint *intBuf;
+  jlong *longBuf;
+  jfloat *floatBuf;
+  jdouble *doubleBuf;
+  jobjectArray objectsArray;
 
-    int i;
-    int SIZE=size;
-    const char *classname="java/lang/Object";
-    jclass clazz = env->FindClass(classname); CE
-    objectsArray = env->NewObjectArray(8,clazz, env->AllocObject(clazz));
+  int i;
+  int SIZE = size;
+  const char *classname = "java/lang/Object";
+  jclass clazz = env->FindClass(classname); CE
+  jobject obj = env->AllocObject(clazz); CE
+  objectsArray = env->NewObjectArray(8, clazz, obj); CE
 
-    arrayArray=(jarray *)malloc(8*sizeof(jarray)); CE
-    arrayArray[BOOL]=env->NewBooleanArray(SIZE); CE
-    arrayArray[BYTE]=env->NewByteArray(SIZE); CE
-    arrayArray[CHAR]=env->NewCharArray(SIZE); CE
-    arrayArray[SHORT]=env->NewShortArray(SIZE); CE
-    arrayArray[INT]=env->NewIntArray(SIZE); CE
-    arrayArray[LONG]=env->NewLongArray(SIZE); CE
-    arrayArray[FLOAT]=env->NewFloatArray(SIZE); CE
-    arrayArray[DOUBLE]=env->NewDoubleArray(SIZE); CE
+  arrayArray = (jarray *)c_malloc(env, 8*sizeof(jarray));
+  arrayArray[BOOL] = env->NewBooleanArray(SIZE); CE
+  arrayArray[BYTE] = env->NewByteArray(SIZE); CE
+  arrayArray[CHAR] = env->NewCharArray(SIZE); CE
+  arrayArray[SHORT] = env->NewShortArray(SIZE); CE
+  arrayArray[INT] = env->NewIntArray(SIZE); CE
+  arrayArray[LONG] = env->NewLongArray(SIZE); CE
+  arrayArray[FLOAT] = env->NewFloatArray(SIZE); CE
+  arrayArray[DOUBLE] = env->NewDoubleArray(SIZE); CE
 
-    for (i=0;i<8;i++)
-    { env->SetObjectArrayElement(objectsArray,i,arrayArray[i]); CE }
+  for (i = 0; i < 8; i++) {
+    env->SetObjectArrayElement(objectsArray, i,arrayArray[i]); CE
+  }
 
-    boolBuf=(jboolean *)malloc(SIZE*sizeof(jboolean));
-    byteBuf=(jbyte *)malloc(SIZE*sizeof(jbyte));
-    charBuf=(jchar *)malloc(SIZE*sizeof(jchar));
-    shortBuf=(jshort *)malloc(SIZE*sizeof(jshort));
-    intBuf=(jint *)malloc(SIZE*sizeof(jint));
-    longBuf=(jlong *)malloc(SIZE*sizeof(jlong));
-    floatBuf=(jfloat *)malloc(SIZE*sizeof(jfloat));
-    doubleBuf=(jdouble *)malloc(SIZE*sizeof(jdouble));
+  boolBuf = (jboolean *)c_malloc(env, SIZE * sizeof(jboolean));
+  byteBuf = (jbyte *)c_malloc(env, SIZE * sizeof(jbyte));
+  charBuf = (jchar *)c_malloc(env, SIZE * sizeof(jchar));
+  shortBuf = (jshort *)c_malloc(env, SIZE * sizeof(jshort));
+  intBuf = (jint *)c_malloc(env, SIZE * sizeof(jint));
+  longBuf = (jlong *)c_malloc(env, SIZE * sizeof(jlong));
+  floatBuf = (jfloat *)c_malloc(env, SIZE * sizeof(jfloat));
+  doubleBuf = (jdouble *)c_malloc(env, SIZE * sizeof(jdouble));
 
-    for (i=0;i<SIZE;i++) {
-    if (i%2 == 0) boolBuf[i]=JNI_TRUE;
-    else boolBuf[i]=JNI_FALSE;
+  for (i = 0; i < SIZE; i++) {
+    if (i % 2 == 0) {
+      boolBuf[i] = JNI_TRUE;
+    } else {
+      boolBuf[i] = JNI_FALSE;
+    }
```

Comment:
Shorten to `boolBuf[i] = (i % 2) ? JNI_TRUE : JNI_FALSE` ?

Reply:
Sure :)

Reply:
but without the implicit boolean :)